homeassistant/components/shelly/light.py

```diff
@@ -178,6 +187,25 @@ def max_mireds(self) -> int:
         """Return the warmest color_temp that this light supports."""
         return int(color_temperature_kelvin_to_mired(self._min_kelvin))
 
+    @property
+    def effect_list(self):
+        """Lights RGB effects list."""
+
+        if self.wrapper.model in RGB_EFFECTS_INDEX:
+            effects_index = RGB_EFFECTS_INDEX.get(self.wrapper.model)
+        else:
+            effects_index = RGB_EFFECTS_INDEX["DEFAULT"]
```

Comment:
Looking at all existing types I think we can have just two sets of lists `BASIC` and `EXTENDED`.
```suggestion
        if self.wrapper.model == 'SHBLB-1`:
            effects = EXTENDED_RGB_EFFECTS
        else:
            effects = BASIC_RGB_EFFECTS
```

Reply:
What if next devices will have more effects ? Just create a 3rd list ? I don't like the idea.

IMHO, is a lot more cleaner and easier to maintain to have a single list and "cut" or "expand" based on a simple index per device type.

Reply:
But that won't be easy if you have to start mixing/matching. I prefer having standalone lists hardcoded per model too.

Reply:
Just curious: what is not easy in maintaining a simple index (that in future firmware will even come from the device itself) compared to maintain both the specific lists and the `if model = xxx` in the code (there will be few) ?

I don't get the point.

Anyway, code updated. Testing.